src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py

```diff
@@ -509,15 +493,25 @@ def _request(self, retry=0, outputFile=None, **kwargs):
     url = url['Value']
 
     # Getting CA file (or skip verification)
-    verify = (not self.kwargs[self.KW_SKIP_CA_CHECK])
-    if verify and self.__ca_location:
+    verify = (not self.kwargs.get(self.KW_SKIP_CA_CHECK))
+    if verify:
+      cafile = Locations.getCAsLocation()
+      if not cafile:
+        gLogger.error("No CAs found!")
+        return S_ERROR("No CAs found!")
       verify = self.__ca_location
 
     # getting certificate
+    # Do we use the server certificate ?
     if self.kwargs[self.KW_USE_CERTIFICATES]:
       cert = Locations.getHostCertificateAndKeyLocation()
+    # CHRIS 04.02.21
+    # TODO: add proxyLocation check ?
```

Comment:
Remove this comment?

Reply:
No no, I really have to do it, but first I need to review the real use case of that